overlord/aspectstate/aspectstate.go

```diff
@@ -26,20 +26,13 @@ import (
 	"github.com/snapcore/snapd/overlord/state"
 )
 
-// Set finds the aspect identified by the account, bundleName and aspect and sets
-// the specified field to the supplied value.
-func Set(st *state.State, account, bundleName, aspect, field string, value interface{}) error {
-	databag, err := getDatabag(st, account, bundleName)
-	if err != nil {
-		if !errors.Is(err, state.ErrNoState) {
-			return err
-		}
-
-		databag = aspects.NewJSONDataBag()
-	}
-
+// SetAspect finds the aspect identified by the account, bundleName and aspect in the
```

Comment:
"in the supplied databag" sounds a bit strange?  

maybe:

SetAspect finds the aspect identified by the account, bundleName and aspect and sets the specified field to the supplied value in the supplied matching databag?